src/features/defi/providers/idle/components/IdleManager/Overview/IdleOverview.tsx

```diff
@@ -218,8 +221,12 @@ export const IdleOverview: React.FC<IdleOverviewProps> = ({
         const tagDetails = idleTagDescriptions[tag as IdleTag]
         return (
           <Flex flexDir='column' px={8} py={4} key={tag}>
-            <Text fontSize='lg' fontWeight='medium' translation={tagDetails.title} />
-            <Text color='gray.500' translation={tagDetails.description} />
+            {tagDetails.title && (
```

Comment:
Is the truthy check here because we expect that `title` can be an empty string? If so, under what circumstances does this occur?